org.freedesktop.Bustle.json

```diff
@@ -0,0 +1,252 @@
+{
+    "app-id": "org.freedesktop.Bustle",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.24",
+    "sdk": "org.gnome.Sdk",
+    /* Use writable-sdk to install Haskell Platform to usr rather than app:
+     * it's only needed to build Bustle, not to run it. All Haskell libraries
+     * and the runtime system are statically linked into the bustle binary.
+     * The only shared library it needs which is not part of the GNOME SDK
+     * is libpcap, which is installed to app.
+     */
+    "writable-sdk": true,
+    "command": "bustle",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=system-bus",
+        "--socket=session-bus",
+        "--share=ipc",
+        /* No support for GtkFileChooserNative in gtk2hs, so the options are:
+         * 1. Use hidden GtkFileChooserButtons and fake clicks on them
+         * 2. Just expose the host FS
+         */
+        "--filesystem=host"
+    ],
+    "build-options": {
+        "strip": false,
+        "no-debuginfo": true
+    },
+    "cleanup": [
+        "*.a", "*.h",
+        "/share/man/man?/pcap*",
+        "/bin/pcap-config"
+    ],
+    "modules": [
+        /* Why not start your day with a 259MB tarball? */
+        {
+            "name": "haskell-platform",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.haskell.org/platform/download/8.0.2/haskell-platform-8.0.2-unknown-posix--full-x86_64.tar.gz",
```

Comment:
Okay. The Haskell Platform bundle appears to only be available for i386 and x86_64 so I guess I will need `{ "skip-arches": ["arm", "aarch64"]}` in any case.